error-monitoring/src/bot.ts

```diff
@@ -32,7 +32,8 @@ export class ErrorIssueBot {
   constructor(
     token: string,
     private repoOwner: string,
-    private repoName: string
+    private repoName: string,
+    private issueRepoName: string | undefined
```

Comment:
As I'm learning more and more TS, I think this should be `issueRepoName?: string`

Reply:
Fixed! I always forget about that, since `string?` is the JSDoc form that Typescript rejects.